### PR TITLE
Book model

### PR DESCRIPTION
**Pull request by juanpablo-rojas**

### CODE REVIEW

File: app/models/book.rb
Comment (MatiasComercio) [inline app/models/book.rb]: Comment is not necessary as this line says that explicitly.

Comment (MatiasComercio) [reply app/models/book.rb]: Same for all the lines below.

File: db/migrate/20180906225330_create_books.rb
Comment (MatiasComercio) [inline db/migrate/20180906225330_create_books.rb]: `default` should not be here, as not specifying any value for model creation will lead to this default, which is indeed a valid not null string, i.e., this `default` is bypassing the `null: false` validation in some way.
You can check this by running `Book.new` (without any arguments) in your rails console.

Comment (MatiasComercio) [reply db/migrate/20180906225330_create_books.rb]: Same for all the lines below.

**Review by MatiasComercio — CHANGES_REQUESTED** [review]

**Comment by MatiasComercio** [comment]
I see through all these last commits that titles are so long that they can be visualized in a single line.
Long commit titles are not ok, and long does not mean descriptive.
A good post for conventions you may read and put in practice since now is [this one](https://chris.beams.io/posts/git-commit/).

There is no convention in Wolox for the following, but at least this is what I find better for my every day work: I love squashing and rebasing. You should use those two tools with caution (and know when merging would be a better choice), but once you master them, git history is so much cleaner and readable that you will be pleased of learning to use them. Let me know if you want to start learning to use them (only if @hdf1986 agrees of course).

File: app/models/book.rb
Comment (hdf1996) [inline app/models/book.rb]: We tend to prefer the oneliner version of this when possible. Something like:
```
validates :genre, :author, :image,..., presence: true
```

**Review by hdf1996 — CHANGES_REQUESTED** [review]

**Review by MatiasComercio — APPROVED** [review]

**Comment by MatiasComercio** [comment]
**PLEASE** wait for @hdf1986 approval. Also, don't forget to read the suggested post for git commits conventions (read my previous comment).

**Review by hdf1996 — APPROVED** [review]

**Pull request merged**
